README.md

```diff
@@ -1 +1,5 @@
-# Code-rewiew-excercise
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Your README is not professional. Remember that Microverse has provided us with the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) to use for all our projects. So, please use it also and then customize it to your project  and make sure all the sections are covered